ChangeLog.rst

```diff
@@ -0,0 +1,62 @@
+Shoop Change Log
```

Comment:
I guess the changelog could live in `doc/` too? Would make it easily browseable via RTFD once it unbreaks.